modules/ROOT/pages/cache-scope-strategy.adoc

```diff
@@ -5,6 +5,8 @@ endif::[]
 
 You can create a caching strategy from a Cache scope properties panel or a Global Elements configuration in Anypoint Studio. After you create a strategy, a Cache component in your flow can reference it.
 
+You can configure the cache size, expire time, and maximum allowed entries by setting these values in the object store that you define or reference from the caching strategy.
```

Comment:
Shouldn't `expire time` be `expiration time` or something similar?

The same change would apply in line 48